pygmt/helpers/utils.py

```diff
@@ -139,7 +140,9 @@ def build_arg_string(kwdict, infile=None, outfile=None):
     Parameters
     ----------
     kwdict : dict
-        A dict containing parsed keyword arguments.
+        A dictionary containing parsed keyword arguments.
+    confdict : dict
+        A dictionary containing GMT's configurations.
```

Comment:
Should we also state that this is optional?
```suggestion
        A dictionary containing configurable GMT parameters.
```

Reply:
The function definition says `confdict=None`. For me, it's clear that `confdict` is optional.

Reply:
Ok